Cargo.toml

```diff
@@ -86,6 +87,9 @@ strip = "symbols"
 [profile.release.package.query-engine]
 strip = "symbols"
 
+[profile.release.package.query-engine-c-abi]
+strip = "symbols"
```

Comment:
Is this the highest level of strip?

Reply:
Yes, `strip = "symbols"` strips both symbols and debuginfo.